nilearn/_utils/niimg_conversions.py

```diff
@@ -176,6 +181,15 @@ def check_niimg(niimg, ensure_ndim=None, atleast_4d=False, dtype=None,
         data will be converted to int32 if dtype is discrete and float32 if it
         is continuous.
 
+    return_iterator: boolean, optional
+        Returns an iterator on the content of the niimg file input
+
+    widlcards: boolean, optional
+        Use wildcards to list matching input images. If multiple files
+        matches and return iterator is set, the returned list of objects
+        is sorted (ascending order), otherwise the first object in the list is
+        returned.
+
```

Comment:
I find this ('return the first object of the list') a strange default behavior. maybe I did not understand.